### ISSUE TITLE
Allow unimplemented interface methods in test classes

### ISSUE THREAD

**Issue by sbcgua** [abaplint/abaplint#819]
![image](https://user-images.githubusercontent.com/15635498/78397008-1225e000-75f9-11ea-9b80-ba092846b8e0.png)

**Comment by larshp** [comment]
in 702, I don't know any way to get around the syntax warnings

for abapGit I do prefer to not have syntax warnings in the source code, its a mess and a lot of work, but PARTIALLY IMPLMENTED, https://syntax.abaplint.org/?filter=interface#/statement/InterfaceDef, only works from 740sp02 and up

![image](https://user-images.githubusercontent.com/5888506/78421307-cc980000-7656-11ea-99e9-a751a284fd05.png)

![image](https://user-images.githubusercontent.com/5888506/78421316-dc174900-7656-11ea-94a2-01237ce75bf9.png)

**Comment by sbcgua** [comment]
hmm, that's interesting ... in my system "Validate" **does not** give the warning for a **test** class. However SCI does. And seems that cannot be disabled by params. That's very unfortunate. I was so happy to find that test classes implicitly support "partial implemented" on 731. Otherwise all methods of the interface must be redefined as empty methods which sucks.

Hmmm, i don't know. What do you think ? The test classes works with partially implemented and at least does not show this error on Ctrl+F3. And this makes sense. Maybe create an option in the rule ?

```abap
class ltcl_da_stub definition final for testing duration short risk level harmless.
  public section.
    interfaces zif_ede_data_accessor. " HAS MANY METHODS AND JUST ONE IMPLEMENTED
    " AND NO WARNINGS FROM CTRL+F3
endclass.
```

**Comment by larshp** [comment]
we can add an option no problem, but I prefer _not_ enabling it for abapGit

long term solution is moving abapGit to 740 syntax, and then automagically downporting it

**Comment by sbcgua** [comment]
hmmm, ok.

**Comment by pokrakam** [comment]
After implementing 76 empty `IF_HTTP...` methods, I would welcome an option to disable this. I can see abaplint can ignore it for test classes but downporting fails.

**Comment by larshp** [comment]
PARTIALLY IMPLMENTED, https://syntax.abaplint.org/?filter=interface#/statement/InterfaceDef, only works from 740sp02 and up
